api/lib/reducers/cosmosV0-reducers.js

```diff
@@ -407,6 +414,7 @@ async function reduceFormattedRewards(
         ? await calculateFiatValue(lunieCoin, fiatCurrency)
         : undefined
       multiDenomRewardsArray.push({
+        id: lunieCoin.denom,
```

Comment:
do we need to specify an extra field `id` or can we specify in GQL what the ID field is?

Reply:
you mean when we do the query? Not sure... you think ID is a special field? Let's google :sweat_smile: